docs/ca/devscu/prepare-a-project-for-spryker-code-upgrader.md

```diff
@@ -42,10 +42,10 @@ The Upgrader supports only projects that run in [cloud environments](/docs/ca/de
 
 For a smooth experience, make sure that your environments meet the minimum technical requirements:
 
-- PHP 8.3+. The Upgrader supports a minimum version of PHP 8.2. PHP 8.1 reached end of life in September 2024, so make sure to update PHP to the [recommended version](/docs/dg/dev/supported-versions-of-php.html). To check if your PHP version is supported, [run the evaluator tool](/docs/dg/dev/guidelines/keeping-a-project-upgradable/run-the-evaluator-tool.html).
+- PHP 8.4+. The Upgrader supports a minimum version of PHP 8.3. PHP 8.2 reached end of life in August 2025, so make sure to update PHP to the [recommended version](/docs/dg/dev/supported-versions-of-php.html). To check if your PHP version is supported, [run the evaluator tool](/docs/dg/dev/guidelines/keeping-a-project-upgradable/run-the-evaluator-tool.html).
```

Comment:
PHP8.2 not really reached it EOL in August. It's not even going to reach it until late next year.
I would re-phrase it in smth like "PHP8.2 is in security support only from the beginning of 2025, so in order to make sure that you can use all new features, please migrate to the recommended version as soon as possible"